tests/integration/worker/test_remote_submission_worker.py

```diff
@@ -0,0 +1,54 @@
+import responses
+import mock
+import os
+import unittest
+
+import scripts.workers.remote_submission_worker as remote_submission_worker
+
+
+class BaseTestClass(unittest.TestCase):
+    def setUp(self):
+        self.download_dir = "/tmp/evalai/"
+        self.download_filename = "file"
+        self.download_path = "{}{}".format(self.download_dir, self.download_filename)
+
+        try:
+            os.makedirs(self.download_dir)
+        except OSError:
+            pass
+
+
+class DownloadAndExtractFileWithProperURL(BaseTestClass):
```

Comment:
@takitsuse can you just create one class named `TestDownloadAndExtractFile` and two separate methods to test both positive and negative cases.